members/ranks/regulars.md

```diff
@@ -2,4 +2,6 @@
 
 Regulars are members who joined and show commitment. Have regular interaction, enthusiasm, motivation, ideas, provide knowledge, educational content or helpful information, support other members, or provide useful links, want to see DOTSAMA and our server succeed.
 
-They have a voice and opinion on every matter that might occur, which is highly considered in most cases. A regular is someone who truly cares about ChaosDAO, our ecosystem and we are sharing the same passion and vision. There is no certain sentence or guideline to this. It comes naturally and its noticeable from most of us without looking something specific. They are true DOTSAMA WARRIORS.
+They have a voice and opinion on every matter that might occur, which is highly considered in most cases. A regular is someone who truly cares about ChaosDAO, our ecosystem and we share the same passion and vision. There is no certain sentence or guideline to this. It comes naturally and its noticeable from most of us without looking something specific. They are true DOTSAMA WARRIORS.
```

Comment:
After ecosystem, "and shares" (eliminate the "we" basically)

Next sentence:

"Although there is no hard and fast rule, most Regulars are recognized for their constant support, information, and value they provide to our organization. They are fully dedicated ChaosDAO members.

Reply:
agreed, I like your version better. previous version had grammar issues and the "ecosystem love, passion, vision" sounds more like DAOist.